<launch>
  <arg name="frequency" default="2" />
  <arg name="ignore_submodels_of" default="" />

  <node name="gazebo2moveit" pkg="gazebo2rviz" type="gazebo2moveit_collision_node.py" args="-f $(arg frequency)" output="screen">
    <param name="ignore_submodels_of" value="$(arg ignore_submodels_of)" type="str" />
  </node>
</launch>
